src/utils/minFee.ts

```diff
@@ -30,3 +30,17 @@ export const calcMinTradableAmountInOwl = ({
   const minFee = minEconomicalViableFeeInOwl.multipliedBy(BUFFER_MULTIPLIER).dividedBy(SUBSIDIZE_FACTOR)
```

Comment:
The `SUBSIDIZE_FACTOR` was recently changed to `5`. Maybe this computation is something that our services can expose as part of a REST API?

Reply:
Yep, in reality we were told we should delete it, so change it to "1".
@Velenir  will soon share one PR (demo) on the what he calls "Traders go away, don't trade, PR" and removes any subsidy

Reply:
> The SUBSIDIZE_FACTOR was recently changed to 5

In production too? this might be an issue for us, since we set the expectation with "10" in the current web
@Velenir we need to make sure the subsidize reduction hit 1.3.0 too